docs/oas/business.yaml

```diff
@@ -2439,6 +2439,109 @@ paths:
+                        comments: []
+                        date: '2024-07-25'
+                        deletionLocked: false
+                        effectiveDate: '2024-07-25T16:17:57.191129+00:00'
+                        email: no_one@never.get
+                        filingId: 150333
+                        inColinOnly: false
+                        isCorrected: false
+                        isCorrectionPending: false
+                        name: incorporationApplication
+                        status: DRAFT
+                        submitter: mocked submitter
+                      incorporationApplication:
+                        contactPoint:
+                          email: no_one@never.get
+                          phone: 123-456-7890
+                        nameRequest:
+                          legalType: BEN
+                          nrNumber: 'NR 1345876'
+                        offices:
+                          recordsOffice:
+                            deliveryAddress:
+                              addressCity: delivery_address city
+                              addressCountry: CA
+                              addressRegion: BC 
+                              postalCode: H0H0H0
+                              streetAddress: delivery_address - address line one
+                            mailingAddress:
+                              addressCity: mailing_address city
+                              addressCountry: CA
+                              addressRegion: BC
+                              postalCode: H0H0H0 
+                              streetAddress: mailing_address - address line one
+                          registeredOffice:
+                            deliveryAddress:
+                              addressCity: delivery_address city
+                              addressCountry: CA
+                              addressRegion: BC
+                              postalCode: H0H0H0
+                              streetAddress: delivery_address - address line one
+                            mailingAddress:
+                              addressCity: mailing_address city
+                              addressCountry: CA
+                              addressRegion: BC
+                              postalCode: H0H0H0
+                              streetAddress: mailing_address - address line one
+                        parties:
+                          - mailingAddress:
+                              addressCity: Victoria
+                              addressCountry: CA
+                              addressRegion: BC
+                              postalCode: V8W 3E6
+                              streetAddress: 200-940 Blanshard St
+                              streetAddressAdditional: ''
+                            officer:
+                              email: severin@example.com
+                              firstName: BCREGTEST Lucille
+                              id: 'p90x2145'
```

Comment:
just wondering, if the id is removed from the request body, will it still shows in the response?

Reply:
Yes